ipatests/test_integration/test_epn.py

```diff
@@ -239,7 +242,7 @@ def cleanupusers(self):
     @pytest.fixture
     def cleanupmail(self):
         """Cleanup any existing mail that has been sent."""
-        for i in self.notify_ttls:
+        for i in range(30):
```

Comment:
Wouldn't max(self.notify_ttls) be more appropriate?

Reply:
That would work too I suppose and be more future proof but we don't twiddle with the configuration and that's all this is doing, ensuring that all old mail gets cleaned up.

Reply:
It's hardcoded there primarily because 30 users are created. This is to ensure that no stray mailboxes are left.

Reply:
Yes, I understand. I wish it was not hardcoded, but it's a simple fix to do if ever we need it. Marking resolved.